src/backend/pluginPlan.test.js

```diff
@@ -116,4 +116,87 @@ describe("src/backend/pluginPlan", () => {
       expect(identity.contractGraph).toBeCalledTimes(0);
     });
   });
+
+  describe("pluginGraphPlan", () => {
+    it("should update discourse graph", async () => {
+      // Given
+      const loaders = mockPluginLoaders();
+      const cache = mockCacheProvider();
+      const ghToken = null;
+      const project = createProject({
+        id: "has-discourse",
+        discourseServer: {serverUrl: "http://foo.bar"},
+      });
+
+      // When
+      const createGraph = pluginGraphPlan(loaders, ghToken, cache);
+      const graph = await createGraph(project);
+
+      // Then
+      const {discourse, github, identity} = loaders;
+      expect(graph).toEqual(discourseGraph());
+      expect(discourse.updateMirror).toBeCalledTimes(0);
+      expect(discourse.createGraph).toBeCalledTimes(1);
+      expect(discourse.createGraph).toBeCalledWith(
+        project.discourseServer,
+        cache
+      );
+      expect(github.updateMirror).toBeCalledTimes(0);
+      expect(github.createGraph).toBeCalledTimes(0);
+      expect(identity.contractGraph).toBeCalledTimes(0);
+    });
+
+    it("fail when missing GithubToken", async () => {
+      // Given
+      const loaders = mockPluginLoaders();
+      const cache = mockCacheProvider();
+      const ghToken = null;
+      const project = createProject({
+        id: "has-github",
+        repoIds: ([{owner: "sourcecred-test", name: "example-github"}]: any),
+      });
+
+      // When
+      const createGraph = pluginGraphPlan(loaders, ghToken, cache);
+      const p = createGraph(project);
+
+      // Then
+      const {discourse, github, identity} = loaders;
+      await expect(p).rejects.toThrow("GithubToken not set");
+      expect(discourse.updateMirror).toBeCalledTimes(0);
+      expect(discourse.createGraph).toBeCalledTimes(0);
+      expect(github.updateMirror).toBeCalledTimes(0);
+      expect(github.createGraph).toBeCalledTimes(0);
+      expect(identity.contractGraph).toBeCalledTimes(0);
+    });
+
+    it("should update github graph", async () => {
+      // Given
+      const loaders = mockPluginLoaders();
+      const cache = mockCacheProvider();
+      const ghToken = ("t000ken": any);
```

Comment:
As discussed here (https://github.com/sourcecred/sourcecred/pull/1571#discussion_r368329145), I think the any-cast is unnecessary code smell.